private/buf/cmd/buf/buf.go

```diff
@@ -343,6 +349,25 @@ func NewRootCommand(name string) *appcmd.Command {
 					},
 				},
 			},
+			// Deprecated shell completion commands (replaced by "buf completion")
+			{
+				Use:        "bash-completion",
+				Deprecated: shellDeprecationMessage("bash"),
+				Hidden:     true,
+				Run:        appcmd.NoAction,
```

Comment:
This is incorrect - we want the commands to still work, just be deprecated. See other examples in this codebase of deprecated commands.